js/gemini.js

```diff
@@ -976,11 +976,8 @@ module.exports = class gemini extends Exchange {
         const code = this.safeCurrencyCode (currencyId, currency);
         const address = this.safeString (transaction, 'destination');
         const type = this.safeStringLower (transaction, 'type');
-        let status = 'pending';
-        // When deposits show as Advanced or Complete they are available for trading.
-        if (transaction['status']) {
-            status = 'ok';
-        }
+        // if status field is available, then it's complete
+        const statusRaw = this.safeString (transaction, 'status');
```

Comment:
We should not lose the `pending` status if a deposit or a withdrawal is not advanced and not complete. Which other statuses are possible?

Reply:
the reason I've removed `pending` is that [withdraw method (`/withdraw` endpoint)](https://docs.gemini.com/rest-api/#withdraw-crypto-funds) doesn't have 'status' endpoint at all,  `parseTransaction` (before we routed `withdrawal` response through it) only served in gemini for [fetchTransactions (`/transfers` endpoint)](https://docs.gemini.com/rest-api/#transfers), which have 'status' only with two possible values: `Advanced` or `Complete`. So, as there was no sign of other, I am not sure we should have `pending`. maybe it could be `cancelled, rejected` or etc. so, why we might say 'pending' in that case?